atlp-rwanda/e-commerce-lydia-32-fn
# Pyament with stripe implementation

### What does this PR do?
This PR implements the integration of Stripe for handling payments on the front-end of the application.
### Description of Task to be completed?

- Set up Stripe integration with the front-end.
- Create a payment form using Stripe Elements.
- Handle payment processing and validation.
- Implement error handling for payment failures.
- Display success messages and handle post-payment actions.
- Ensure secure handling of payment information.

### How should this be manually tested?

- Start the front-end application using yarn run dev.
-Navigate to the checkout page and initialize payment.
-Fill in the payment form with test card details provided by Stripe (e.g., 4242 4242 4242 4242 for a successful transaction).
-Submit the form and verify that the payment is processed successfully.
-Verify that the success message is displayed and any post-payment actions are executed.

### What are the relevant pivotal tracker stories

https://www.pivotaltracker.com/story/show/187570840
